LICENSE.txt

```diff
@@ -1,25 +1,18 @@
-Copyright (c) 2012-15, Members of the Raising the Floor Consortium.
+Copyright (c) 2012-17
+Members of the Raising the Floor Consortium
 All rights reserved.
 
-Redistribution and use in source and binary forms, with or without
-modification, are permitted provided that the following conditions are met:
+This is a test version of GPII. You will install the software on your computer
+for testing, and uninstall it at the end of the test.
 
- * Redistributions of source code must retain the above copyright notice, this
-   list of conditions and the following disclaimer.
- * Redistributions in binary form must reproduce the above copyright notice,
-   this list of conditions and the following disclaimer in the documentation
-   and/or other materials provided with the distribution.
- * Neither the name of OCAD University nor the names of its contributors may
-   be used to endorse or promote products derived from this software without
-   specific prior written permission.
+Please do not share it or give it to anyone else, because this just a test
```

Comment:
@colinbdclark Yes, this is targeted to master because currently doesn't exists the HST branch in the original repository.

About the two licenses, in reality is probably a bad naming. The main LICENSE file is the software license of all the installer software (That probably will be erased if we finally moved this into the windows repository) and the other License.rtf file is the content that is injected in the EULA page of the installer.

The main idea is to work in the separated branch for HST that will be merged in the future into master, as probably GPII-App will be the initiator of any Windows installation.